NeuralAmpModeler/NeuralAmpModelerControls.h

```diff
@@ -784,7 +784,7 @@ class NAMSettingsPageControl : public IContainerBaseWithNamedChildren
     {
       IControl::SetValueFromDelegate(normalizedValue, valIdx);
       const std::string s = ConvertToString(normalizedValue);
-      OnTextEntryCompletion(s.c_str(), valIdx);
+      SetStr(s.c_str());
```

Comment:
Hmm...I wonder if the root cause was that the argument to `SetDirty()` is `true` but should be false.

I need to double-check this. I'm worried whether not calling `SetDirty()` will cause something not to react correctly.

Thanks for your patience; I'll just want to be sure.